libraries/Microsoft.Bot.Builder.Dialogs.Debugging/vscode-dialog-debugger/.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
+	"version": "0.2.0",
+	"configurations": [
```

Comment:
>configurations [](start = 2, length = 14)

this file allows you to "start VCode, open this extension folder, and then start debugging to start a new instance of VSCode with the extension loaded" so you can use the debugger or diagnose any issues (without installing the extension into the user's directory).